src/components/list/ListPage.tsx

```
@@ -4,100 +4,55 @@ import { useEffect, useRef, useState } from "react";
+      if(sort === newSort) return;
+      setCursorValue(null);
+      setLastProjectId(null);
+      setProjects([]);
+      setSort(newSort);
+  }
 
-    const handleSortClick = (sort: "new" | "deadline" | "recommend" | "view" | "achievement") => {
-        setSort(sort);
-    }
+  const loadProjects = () => {
+    apiCall(url + nextPageQueryParam, "GET").then(({ data }: { data: any }) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)